libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -224,6 +224,11 @@ size_t WiFiClient::write(const uint8_t *buf, size_t size)
     return res;
 }
 
+size_t WiFiClient::write_P(PGM_P buf, size_t size)
+{
+    return write(buf, size);
+}
+
```

Comment:
> I can agree to mark them deprecated and remove at later point.

What approach would you suggest library authors take when it comes to writing code which supports both the ESP8266 and the ESP32?